LCM/dsc/engine/EngineHelper/EngineHelperInternal.h

```diff
@@ -86,9 +86,9 @@
     EH_Check(MI_RESULT_OK == (miResultExpression))
 
 /* unwind block */
+#pragma GCC diagnostic ignored "-Wunused-label"
```

Comment:
If we are ignoring werror on DSC this is fine but it might generate more warnings on other compilers.
For Visual Studio : `If the compiler finds a pragma that it does not recognize, it issues a warning and continues compilation.`